Test/FMI3/parser_test_xmls/default_experiment/defined/modelDescription.xml

```diff
@@ -1,18 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<fmiModelDescription
-  fmiVersion="3.0"
-  modelName="myModelName"
-  instantiationToken="myInstantiationToken"
-  description="myDescription"
-  author="myAuthor"
-  generationTool="myGenerationTool"
-  generationDateAndTime="2002-05-30T09:00:00"
-  version="myModelVersion"
-  variableNamingConvention="structured"
-  numberOfEventIndicators="3">
-
-<ModelExchange
-  modelIdentifier="myModelIdentifier" />
+<fmiModelDescription fmiVersion="3.0" modelName="" instantiationToken="" variableNamingConvention="structured">
+<ModelExchange modelIdentifier="id"/>
```

Comment:
Very many tests specified `numberOfEventIndicators` which now results in warnings. So when fixing that I decided to remove all optional attributes as well.